docusaurus/docs/create-a-plugin/extend-a-plugin/add-resource-handler.md

```diff
@@ -18,7 +18,7 @@ You can add a resource handler to your data source backend to extend the Grafana
 
 ## Uses of resource handlers
 
-The primary way for a data source to retrieve data from a backend is through the [query method](./add-query-editor-help.md). But sometimes your data source needs to request data on demand; for example, to offer auto-completion automatically inside the data source’s query editor.
+The primary way for a data source to retrieve data from a backend is through the [query method](../../how-to-guides/data-source-plugins/add-query-editor-help). But sometimes your data source needs to request data on demand; for example, to offer auto-completion automatically inside the data source’s query editor.
```

Comment:
This link doesn't actually make sense, could either remove or instead point to `/tutorials/build-a-data-source-plugin#define-a-query`

Reply:
fixed this issue in commit https://github.com/grafana/plugin-tools/pull/976/commits/6cf5265b50cc92a0d22032e8967bed33327ce5ed